README.md

```diff
@@ -182,6 +185,14 @@ The following parameters are available for the hiera class:
   Arbitrary YAML content to append to the end of the hiera.yaml config file.  
   This is useful for configuring backend-specific parameters.  
   Default: `''`
+* `provider`  
+  Which provider to use to install hiera-eyaml. Can be `puppetserver_gem` (PE 2015.x), `pe_puppetserver_gem` (PE 3.7 and 3.8), `pe_gem` (PE pre-3.7), `puppet_gem` (FOSS using puppet's gem), or `gem` (FOSS using system's gem)
```

Comment:
Yes, but there is no way to detect a foss puppetserver master vs. a foss ruby puppet master, so I default to just using puppet_gem. It's suboptimal, but I can't figure out how to fix it...